api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -593,12 +593,12 @@ public UserRecentWorkspace updateRecentWorkspaces(
   }
 
   @Override
+  @Transactional
```

Comment:
Do we need a test case that proves rollback when this method fails?

Reply:
I've had trouble getting a unit test to replicate the exception I was seeing that prompted me to add an `@Transactional` in the first place.

Reply:
Yeah, these types of test cases are usually tough to implement. Thanks for trying.